### DISCUSSION TITLE
How to import SRFI in REPL?

### DISCUSSION THREAD

**Discussion by jcubic** [gambit/gambit#894] [Q&A] [answered]
If I try to import SRFI-1 got this error:

```scheme
(import (srfi 1))
*** ERROR IN (stdin)@16.1-16.19 -- Ill-formed special form
```

I can't find any documentation how to use the REPL or how to import a library.

**Comment by gambiteer** [comment, marked as answer]
Maybe you're using an older version that doesn't support that notation.  I get
```
heine:~/programs/gambit/gambit> gsi
Gambit v4.9.5-124-g6d1a9a9b

> (import (srfi 1))
> (filter odd? '(0 1 2 3 4 5 6))
(1 3 5)
```
Which version of Gambit are you using?

**Reply by jcubic** [reply]
Ah, it seems that I had two versions latest from `git` and some old that in `/usr/local/Gambit/bin/gsi`

Running `/usr/bin/env gsi` run version v4.9.5-93-gfe72401d and import do work.
